db/postGresSchema.sql

```diff
@@ -1,21 +1,27 @@
-DROP DATABASE IF EXISTS opentablereservations;
-
-CREATE DATABASE opentablereservations;
-
-\c opentablereservations;
-
-CREATE TABLE IF NOT EXISTS restaurants (
-  id SERIAL UNIQUE PRIMARY KEY,
-  restaurantName text NOT NULL
-);
-
-CREATE TABLE IF NOT EXISTS reservations (
-  id SERIAL UNIQUE PRIMARY KEY,
-  restaurantID INTEGER REFERENCES restaurants(id),
-  dateToReserve varchar(6) NOT NULL,
-  timeToReserve varchar(4) NOT NULL,
-  partySize varchar(2) NOT NULL
-);
-
-/* run psql -U YOUR_USER_NAME -W < db/postGresSchema.sql
-from the root directory and enter your password for the prompts */
\ No newline at end of file
+DROP DATABASE IF EXISTS opentablereservations;
+
+CREATE DATABASE opentablereservations;
+
+\c opentablereservations;
+
+CREATE TABLE IF NOT EXISTS restaurants (
+  id SERIAL UNIQUE PRIMARY KEY,
+  restaurantName text NOT NULL
+);
+
+CREATE TABLE IF NOT EXISTS reservations (
+  id SERIAL UNIQUE PRIMARY KEY,
+  restaurantID INTEGER REFERENCES restaurants(id),
+  dateToReserve varchar(6) NOT NULL,
+  timeToReserve varchar(4) NOT NULL,
+  partySize varchar(2) NOT NULL
+);
+
+COPY restaurants(id, restaurantName) FROM 'C:/Users/michael Lau/Service-Mike/db/restaurants.csv' (FORMAT CSV);
```

Comment:
Is there a way to use relative paths in a .sql file?

Reply:
unfortunately not